mobile-widgets/qml/HintsTextEdit.qml

```diff
@@ -16,7 +16,7 @@ TextField {
 	}
 	onFocusChanged: {
 		focus != focus
-		frame.shouldShow = focus
+		frame.shouldShow = false
```

Comment:
I don't think I fully understand what the commit message is telling me.
With this we never show the drop down? That doesn't make sense...